src/integrations/BingAds/browser.js

```diff
@@ -19,7 +19,7 @@ class BingAds {
           let o = {
             ti: this.tagID,
           };
-          (o.q = w[u]), (w[u] = new UET(o));
+          (o.q = w[u]), (w[u] = w.UET ? new UET(o): logger.error('Blocker faced in loading UET class. Probable solution is to unblock domain bat.bing.com in your browser'));
```

Comment:
```suggestion
          (o.q = w[u]), (w[u] = w.UET ? new UET(o): logger.error('UET function is not loaded'));
```
No need to share the error message in detail.

Reply:
changed the code a bit. Can you check once?